src/test_helpers/mod.rs

```diff
@@ -464,11 +457,11 @@ impl PrivateApi for TestKitApi {
 
 fn validate_table_proof(
```

Comment:
Off-topic: I've been thinking about adding the corresponding functionality into `IndexProof`. Would it make sense?

Reply:
Yes, I think that whould be useful in service tests.